test-site/config/global_config.json

```diff
@@ -3,6 +3,7 @@
   // "token": "<REPLACE ME>", // The auth token to access Answers experience.
   "apiKey": "2d8c550071a64ea23e263118a2b0680b", // The answers api key found on the experiences page. This will be provided automatically by the Yext CI system
   // "experienceVersion": "<REPLACE ME>", // the Answers Experience version to use for API requests. This will be provided automatically by the Yext CI system
+  //"environment": "<REPLACE ME>", // The environment to run on for this Answers Experience. (e.g. 'production' or 'sandbox')
```

Comment:
space after double after the double slash

Reply:
yea